treeherder/model/error_summary.py

```diff
@@ -28,6 +28,71 @@
 PREFIX_PATTERN = r"^(TEST-UNEXPECTED-\S+|PROCESS-CRASH)\s+\|\s+"
 
 
+class MemDBCache:
+    keyroot = ""
+
+    def __init__(self, keyroot):
+        self.keyroot = keyroot
+
+    def get_cache(self):
+        lcache = {}
+        keys = self.get_cache_keys()
+        # copy db cache to memory cache
+        if not keys or not cache.get(keys[0]):
```

Comment:
`not keys or` does nothing here. This should be `if keys and not cache.get(keys[0]):`

Reply:
thanks, this makes sense